x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -345,7 +336,7 @@ const TGridStandaloneComponent: React.FC<TGridStandaloneProps> = ({
 
   return (
     <InspectButtonContainer>
-      <StyledEuiPanel data-test-subj="events-viewer-panel" $isFullScreen={false}>
+      <AlertsTableWrapper data-test-subj="events-viewer-panel">
```

Comment:
to get the failing test to pass, you just need this data-test-subj to appear on the wrapper or rendered on a separate element in this component

Reply:
Thanks 👍  I was helping Katrin get these last failing CI steps to pass. For my own sanity could you explain why I had to bump this to the outer wrapper? (Which is passing).

![Screenshot 2021-08-17 at 17 06 12](https://user-images.githubusercontent.com/471693/129762383-ea6f66dc-6caf-41bc-8175-c86aff44aed0.png)

With the original structure the element is still there, in the DOM, but doesn't get returned?

Reply:
From the screenshot, no I cannot, that looks like it should pass 😢

Reply:
Super weird! But thanks for looking.